grafana/overlays/nerc-ocp-infra/patches/logging-grafana-patch.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: Patch
+metadata:
+  name: logging-grafana-patch
+  namespace: grafana
+spec:
+  serviceAccountRef:
+    name: patch-operator-tokenrequest
+  patches:
+    test-patch:
```

Comment:
do you think we should have a more descriptive name?

Reply:
Oops forgot to update the name here

Reply:
Should be logging-grafana-patch as on line 4

Reply:
@dystewart okay, once that's updated I can approve this PR.

Reply:
@dystewart once changed this should be ready for a second approval.